.gitignore

```diff
@@ -3,3 +3,4 @@
 psqldump
 .DS_Store
 cantusdb/turtle/out_rdf.ttl
+.history/*
```

Comment:
This should be removed from the PR. There is no explanation of why it is here and is not referenced by any other file in the PR.

Reply:
That's fine. Can you add to the description of this PR something like: "Adds the .history directory (used by the ______ VSCode extension to .gitignore"

Reply:
This is changed in other PRs as well. I will also document in other PRs.